### PR TITLE
streamingccl: add more metrics to LDR Metrics page

### PR DESCRIPTION
**Pull request by msbutler**
This patch adds a few more metrics to the ldr db console page. The page now
displays:
- Logical Bytes Received i.e. sum size of KVs that were decoded from stream
  client (one plot per-node)
- Row Updates Applied i.e. number of kvs that were decoded and "applied"
  (single plot of sum across nodes)
- Age of Applied Row Updates i.e. commit-to-commit/age of mvcc ts (one plot per
  p50/p90/p99 taking max across nodes)
- Maximum Age of Update Pending Replication i.e. now - frontier ts (one plot w/
  min across nodes)
- Batch Application Processing Time p50 and p99 of the write queries (two plots
  per node)

Epic: none

Release note: none

### CODE REVIEW

**Comment by dt** [comment]
Here's what I think is wishlist:

* row updates sent num kvs that went into batches that eventStream sent  (single plot of sum across nodes)
* row updates applied i.e. number of kvs that were decoded and "applied" (single plot of sum across nodes)
* conflicts resolved
* failed batches/dlq's
* age of applied applied row updates (one plot per p50/p90/p99 max across nodes)
* maximum age of update pending replication i.e. now - frontier ts (one plot w/ min across nodes)
* megabytes sent (one plot per-node)
* megabytes received (one plot per-node)
* batches applied ie. number of update/insert/delete queries run (one plot per node)
* batch application processing time p50 and p99 (two plots per node)
* emission queue delay (time spent waiting for `eventStren.Next() calls`

Now obviously _when debugging_ some of these would be more useful per-node, i.e. if you're looking for an imbalance then the number of KVs received would be useful to see per-node, or the p99 of batches per node or even per processor, but I think some combo of the custom charts page and our vtables can be what we use for _involved_ debugging, and the above should give us a good dashboard with which to scan and see if there is something that needs more digging.

**Comment by ajstorm** [comment]
In addition to David's excellent list, here's my wishlist:

* `p50/p95/p99 conflicts` - i.e. split out the conflicting row latency from the non-conflicting rows, as I'd imagine they will have quite different performance, at least for some UDF constructions
*  `replanning events` - i.e. the amount of times we had to replan the job
* `buffer consumption rangefeed source` - amount of rangefeed buffer we're consuming at the source
* `buffer consumption producer` - amount of data buffered at the producer side

File: pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/logicalDataReplication.tsx [hunk 1]
```diff
@@ -9,46 +9,103 @@
...
+
+    <LineGraph
+    title="Event Commit Latency: 50th percentile"
+    isKvGraph={false}
+    tenantSource={tenantSource}
+    tooltip={"The 50th percentile in the difference in INSERT, UPDATE, and DELETE commit times between the source cluster and the destination cluster"}
+    showMetricsInTooltip={true}
+  >
+    <Axis units={AxisUnits.Duration} label="latency">
+      {map(nodeIDs, node => (
```
[the remark this answers is not in the document]
Comment (msbutler) [reply pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/logicalDataReplication.tsx]: Done.

**Review by dt — APPROVED** [review]
Let's capture the wishlist in an issue and check off the ones we have as they merge

**Comment by msbutler** [comment]
Tftr!

bors r+

**Pull request merged**